lib/processor.js

```diff
@@ -39,10 +39,16 @@ exports.work = (audioEvent) => {
 // upload mp3 and make copies as needed
 exports.uploadAndCopy = (file) => {
   let dest = `${file.path}/${file.name}`;
+
+  // set content type - translate audio/mp* to audio/mpeg
   let type = file.contentType;
   if (!type && file.format) {
      type = `audio/${file.format}`;
   }
+  if (type && type.match(/audio\/mp./)) {
```

Comment:
why not specifically mp3?
Will this mess up things for mp4 or other types of files?
```
type: audio/mp4, file extensions: .mp4 .m4a
type: audio/mpeg, file extensions: .mp1 .mp2 .mp3 .mpg .mpeg
```